auth/index.html

```diff
@@ -213,6 +213,9 @@ <h2>Message log</h2>
        * @returns {Uint8Array}
        */
       var uint8arrayFromHexString = function(hexString) {
+        if (!hexString || hexString.length % 2 != 0 || !/^[0-9A-Fa-f]+$/.test(hexString)) {
```

Comment:
nit: we could move the regex into a variable so it's slightly more readable

```
var hexRegex = /^[0-9A-Fa-f]+$/;
if (!hexString || hexString.length % 2 != 0 || !hexRegex.test(hexString)) {
...
}
```

Reply:
missed opportunity to name this `regHex` 😅